e2e/e2etests/test_precompiles_bank.go

```diff
@@ -0,0 +1,119 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	"github.com/zeta-chain/node/precompiles/bank"
+)
+
+func TestPrecompilesBank(r *runner.E2ERunner, args []string) {
```

Comment:
This test should be refactored to reset values at the beginning or you relative values, so that it still works if we run it for a second time( for the TSS migration test )

Reply:
I included an allowance reset in 4d0a57bb9424dd94ca93a55ff5ff86bb49cab3cd.
Other than that it should be ok to run this e2e twice, as the balances are reverted at the end of the test.